src/coreclr/jit/assertionprop.cpp

```diff
@@ -164,6 +164,22 @@ bool IntegralRange::Contains(int64_t value) const
         case GT_CAST:
             return ForCastOutput(node->AsCast());
 
+#ifdef FEATURE_HW_INTRINSICS
+        case GT_HWINTRINSIC:
+            if (HWIntrinsicInfo::IsXCNTIntrinsic(node->AsHWIntrinsic()->GetHWIntrinsicId()))
+            {
+                return {SymbolicIntegerValue::Zero, SymbolicIntegerValue::IntMax};
+            }
+            break;
+#endif // FEATURE_HW_INTRINSICS
+
+        case GT_INTRINSIC:
```

Comment:
Re: `NI_System_Numerics_BitOperations_PopCount` - handling `GT_INTRINSIC` is not necessary.

Reply:
Updated in b72739a52fc7b188720627bab0a814a45a4fd09a